### PR TITLE
Styling added

### PR DESCRIPTION
**Pull request by redmarverick**
- Styles added using tailwindcss
- Added an extra input for the author so the website works as intended

Please ignore the update delay, thats a problem im going to solve later, i just need this to finish my week, as i only have 2 hours please aprove as soon as possible, if you have any suggestion, please add to optional and i will do it.

**TO DEPLOY USE:
npm run build
npm run dev**

### CODE REVIEW

File: src/components/BookForm.jsx [hunk 1]
```diff
@@ -26,28 +26,61 @@ const BookForm = () => {
...
+          placeholder="Book title"
+          className="w-1/3 h-12 mr-4 pl-3 pr-14 rounded-md border border-gray-300 bg-white"
         />
         <input
           type="text"
           value={author}
           onChange={(e) => setAuthor(e.target.value)}
-          placeholder="Enter book author"
+          placeholder="Author"
+          className="w-1/3 h-12 mr-4 pl-3 pr-14 rounded-md border border-gray-300 bg-white"
```
Comment (Munsa1) [inline src/components/BookForm.jsx]: - [x] The point of this milestone is to follow the given [zeplin](https://zeplin.io/) design, this includes the font-family used in the original design. Kindly apply this to your project as well. Please note that there are two different fonts applied throughout this project, `RobotoSlab` and `Montserrat`

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @redmarverick, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Nice Design :heavy_check_mark: 
Used A CSS FrameWork :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Going forward kindly use the imperative tense for your commit messages as stated in these [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.jsx (hunk 1 above)
Comment (redmarverick) [reply src/components/BookForm.jsx]: done

**Review by Munsa1 — APPROVED** [review]
Hi @redmarverick, 
congratulations-1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**